### PR TITLE
Added troubleshooting section on how to determine which is breaking the cache

### LINKED ISSUE #4821: WordPress and PHP Sessions Doc Update add troubleshooting caching and sessions issue
Re: [WordPress and PHP Sessions](https://pantheon.io/docs/wordpress-sessions/)

Priority: Low

## Issue Description:
There are lot of issues where SESSION cookie is breaking the cache

## Suggested Resolution
It would be helpful for users if the common steps in troubleshooting the issue, documented suggested steps for them to determine which plugin/thems is causing the issue

### PR DESCRIPTION
**Pull request by carl-alberto**
Closes #4821

## Effect
PR includes the following changes:
- Added troubleshooting section on how to determine which is breaking the cache

## Remaining Work
- [x] Technical review
- [ ] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Archive from **Done** in [Main Project](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by jfussion** [comment]
@carl-alberto I suggest for a reverse approach. Disabling all the plugins first will allow the page to be cached. Varnish checker will still get a cached version of the page until that cache expires.

The reverse approach is to disable plugin one by one then check the page if it starts to be cached. The last plugin disabled is probably the culprit

**Comment by alexfornuto** [comment]
I've assigned this to both of you to discuss and figure out the best solution. don't worry about the merge conflicts, I'll handle that later.

**Comment by carl-alberto** [comment]
Thanks @alexfornuto 

Please review the revision @jfussion 

Ty!

File: source/_docs/wordpress-sessions.md [hunk 1]
```diff
@@ -24,25 +24,20 @@ Plugins with session-using code are relying on PHP's default session manager, wh
...
 Symptoms of this issue shows when the header is inspected, you will see that the session cookie is always set on every page load eg. 
 
 ```
 Set-Cookie: SESS1234XXXXXXXXXXXXXX path=/; domain=.example.pantheonsite.io; HttpOnly
 ```
 
-The best way to determine which plugin or theme is not caching your site is by using this [tool](https://varnishcheck.pantheon.io/) with each of the steps until you determine which component is breaking the cache:
+The best way to determine which plugin or theme is not caching your site is by using this [tool](https://varnishcheck.pantheon.io/) or by inspecting the headers using `curl -sI example.com` with each of the steps until you determine which component is breaking the cache:
 1) To check if it is the theme that is breaking the cache, use your default theme like twentynineteen and check with the tool.
 2) To check if it is any of the plugin is breaking the cache, disable the plugins and turn it on one by one while checking with the tool if it breaks the cache. Do not forget to clear the cache from the Pantheon dashboard everytime that you disable a plugin.
```
Comment (jfussion) [inline source/_docs/wordpress-sessions.md]: @carl-alberto Can we reverse the approach here? Disable the plugin one by one while checking with the tool if the page starts to cache again. The last plugin that was disabled is probably the culprit why the page is not being cached.

Comment (carl-alberto) [reply source/_docs/wordpress-sessions.md]: Sounds good @jfussion, revised the TS as per your suggestion, please review. Thanks!

**Comment by alexfornuto** [comment]
@carl-alberto please review my copy edits.

**Comment by alexfornuto** [comment]
Moving on...

**Pull request merged**